__tests__/index.test.js

```diff
@@ -631,3 +630,58 @@ describe("POST endpoints", () => {
     });
   });
 });
+describe("DELETE endpoints", () => {
+  describe("DELETE /api/comments/:comment_id", () => {
+    it("returns a status code of 204", () => {
+      return request(app).delete("/api/comments/1").expect(204);
+    });
+    it("returns an empty body", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+        .then(({ body }) => {
+          expect(Object.keys(body).length).toBe(0);
+        });
+    });
+    it("removes a comment from the db based on the provided id, if suitable", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+        .then(({ body }) => {
+          return db.query("SELECT * from comments WHERE comment_id = 1");
+        })
+        .then(({ rowCount }) => {
+          expect(rowCount).toBe(0);
+        });
+    });
+    it("works with repeated calls", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+        .then(({ body }) => {
+          return db.query("SELECT * from comments WHERE comment_id = 1");
+        })
+        .then(({ rowCount }) => {
+          expect(rowCount).toBe(0);
+          // make another api call
+          return request(app).delete("/api/comments/1").expect(204);
+        })
+        .then(({ body }) => {
+          return db.query("SELECT * from comments WHERE comment_id = 1");
```

Comment:
This should be handled as an api request and not just a direct query. What kind of behaviour should a client get if they try to delete a comment which doesnt exist/has been deleted?